### PR TITLE
Cara/login logout/wed 68

### PR DESCRIPTION
**Pull request by caramcc**
My actual changes: https://github.com/ID8-neu/wediko-secret-snowman/compare/cara/login-logout/WED-68?expand=1#diff-21497849d8f00507c9c8dcaf6288b136R12

and https://github.com/ID8-neu/wediko-secret-snowman/compare/cara/login-logout/WED-68?expand=1#diff-55c5b7aecfb519d0e4880eaf2788eb6eR5

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
   root 'welcome#index'
 
+  devise_scope :user do
+    get '/login' => 'devise/sessions#new'
+    post '/logout' => 'devise/sessions#destroy'
```
Comment (ThePletch) [inline config/routes.rb]: shouldn't this be `delete`?

**Comment by ThePletch** [comment]
one small comment, otherwise :shipit:

**Comment by jonathanchiu** [comment]
Should we be incorporating the login view along with this as well?

**Comment by ThePletch** [comment]
@DerekWChan do we have a mockup/template for the view?

**Comment by DerekWChan** [comment]
Hey, I posted the mockup/template for the Login Page to Slack. I don't have GitHub setup on my computer atm, sorry. I will try to get it setup this weekend so I don't have to edit the front end stuff by proxy.

**Comment by ThePletch** [comment]
this is blocking people, so let's worry about getting the view put in later and just merge this

**Pull request merged**